config/config.go

```diff
@@ -468,6 +476,11 @@ func NewConfig() (c *Config) {
 				Enabled:                true,
 				IsCore:                 false,
 				NamespaceLabel:         "kubernetes_namespace",
+				Prometheus: PrometheusConfig{
+					ThanosProxy: ThanosProxy{
+						Enabled: false
+					}
+				}
```

Comment:
This should fix the linter error that is causing the build to fail:
```suggestion
						Enabled: false,
					},
				},
```